### PR TITLE
remove myself from a few modules

### PR DESCRIPTION
**Pull request by pmcollins**

### CODE REVIEW

**Review by atoulme — APPROVED** [review]
Changes look good, no need for changelog entry, please remove and we can get this in. Thank you for all your help!

File: .chloggen/codeowners-pmc.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
```
[the remark this answers is not in the document]
Comment (pmcollins) [reply .chloggen/codeowners-pmc.yaml]: ah, ok. fixed.

**Review by jpkrohling — COMMENTED** [review]
@dmitryax, could you please review this as well, as you are effectively becoming the code owner of some components as part of this change?

**Comment by atoulme** [comment]
My plan is to advertise internally and externally and bring more codeowners to those components. Please let it be known far and wide!

**Review by codeboten — APPROVED** [review]

**Review by dmitryax — APPROVED** [review]

**Pull request merged**